### ISSUE TITLE
Unknown "V1" taxa in abundance taxa plot

### ISSUE THREAD

**Issue by IdanBanson** [compbiomed/animalcules#69]
I keep seeing the presence of an unknown taxa with the name "V1" anytime I construct my taxa bar plots. However there is nothing in my taxanomy.tsv file that traces back to this "V1" taxa and it seems to be the most abundant in my sample as well. 
![newplot](https://github.com/wejlab/animalcules/assets/147667333/d7cb8690-ba9d-496e-8f57-e1000015cc5c)

**Comment by jasonzhao0307** [comment]
I'm wondering if your uploaded data match the data format as in the instruction below?
![](https://github.com/wejlab/animalcules-docs/blob/master/images/count2.PNG?raw=true)

**Comment by wevanjohnson** [comment]
Yes, is it possible that one of your microbes doesn’t have proper annotation or is missing annotation? V1 is what R assigns to missing column namesOn May 1, 2024, at 5:00 PM, Yue Zhao ***@***.***> wrote:﻿
I'm wondering if your uploaded data match the data format as in the instruction below?

**Comment by IdanBanson** [comment]
@wevanjohnson Your explanation seems to match what I am seeing in my taxonomy table. Some of my microbes only resolved at the phylum level and hence do not have any designated class, order, family, species, etc... To confirm that I realised from the images below, I do not get any V1 data when I plot with phylum level since all my microbes have phyla names however I start to see "V1" from class levels where the missing spaces start to appear. 
![Screenshot from 2024-05-01 18-35-34](https://github.com/wejlab/animalcules/assets/147667333/35877f4e-6c95-4280-9c11-599c6533c3da)
![Screenshot from 2024-05-01 18-34-40](https://github.com/wejlab/animalcules/assets/147667333/46ff2818-b61c-409e-8c57-de8eae3a99a5)

**Comment by IdanBanson** [comment]
I guess with that resolved my final question is that does the V1 data affect other subsequent analysis such as Alpha diversity metrics?

**Comment by wevanjohnson** [comment]
Hmm, this is a tough question. Microbe name doesn’t impact the alpha/beta diversity calculation, so it should calculate fine. But including it, means that you are including higher-level taxa data in your calculation of lower-level diversity, which seems wrong. But excluding it also seems like you are reducing your diversity—maybe you should exclude it and say that there are some missing taxa? I am assuming you are using QIIME2 or DADA2 for your taxonomic classification? We just published a paper showing that metagenomic methods do a better job at taxonomy assignment that DADA2. This might help a bit with your issue (remove some of the higher level assignments): nature.comOn May 1, 2024, at 5:43 PM, IdanBanson ***@***.***> wrote:﻿
I guess with that resolved my final question is that does the V1 data affect other subsequent analysis such as Alpha diversity metrics?

**Comment by IdanBanson** [comment]
Yes, The taxonomic classifications were done with QIIME2 and the denoising step was done using Deblur. Unfortunately this is Fungal ITS2 amplicon sequencing data and hence my options for bioinformatic analysis gets really narrowed down. Thank you for your suggestions and the article recommendation (Odom et al., (2023) right? ). I will suggest them to my team tomorrow and decide on what to do next with the data.
